### PR TITLE
Add permission fetching

### PR DESCRIPTION
**Pull request by rudigiesler**
Add an endpoint to retrieve the permissions that a user has, by giving a token for that user.

### CODE REVIEW

**Comment by rudigiesler** [comment]
@justinvdm Ready for review.

File: authapi/urls.py [hunk 1]
```diff
@@ -40,4 +40,7 @@
 urlpatterns = [
     url(r'^', include(router.urls)),
     url(r'^/user/tokens/$', views.TokenView.as_view(), name='create-token'),
+    url(
+        r'^/user/permissions/$', views.UserPermissionsView.as_view(),
+        name='get-user-permissions')
```
Comment (justinvdm) [inline authapi/urls.py]: Didn't we decide to have a `/user` endpoint that contains the permissions as a property (and the `admin` property)? Or are we going to add that separately?

**Comment by justinvdm** [comment]
One minor comment, looks good!

File: authapi/urls.py (hunk 1 above)
Comment (rudigiesler) [inline authapi/urls.py]: Ah right, yes we did. We just haven't updated the docs yet. Let me update it and change this.

Comment (justinvdm) [inline authapi/urls.py]: Ah, woops, said I'd update that in docs and then forgot, thanks :)

**Comment by rudigiesler** [comment]
@justinvdm I've changed the URL to `/user` and updated the docs. Ready for review again.

**Comment by justinvdm** [comment]
:+1: when travis is happy

**Pull request merged**